pulser-core/pulser/backend/config.py

```diff
@@ -145,6 +148,9 @@ class EmulationConfig(BackendConfig, Generic[StateType]):
     # Whether to warn if unexpected kwargs are received
     _enforce_expected_kwargs: ClassVar[bool] = False
 
+    _state_type: ClassVar[Type[State]]
+    _operator_type: ClassVar[Type[Operator]]
```

Comment:
Take note of this new class var @a-quelle @kbidzhiev . In specific subclasses, it should be defined so that the deserialized observables come with states and observables in the supported type.

In `MPSConfig`, I would expect to see:
```
_state_type = MPS
_operator_type = MPO
```